mandatory/4-tax.js

```diff
@@ -17,7 +21,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let priceAddTax =calculateSalesTax(price);
```

Comment:
Really nice to see you using a function you've already created inside another function!